src/it/resources/com/google/checkstyle/test/chapter4formatting/rule488numericliterals/InputNumericLiterals.java

```diff
@@ -74,18 +74,18 @@ private long foo() {
                   return 666l + 666L; // violation 'Should use uppercase 'L'.'
                 }
 
-                private void processUpperEll(long aLong) {
+                private void processUpperEll(long x) {
                   long bad =
                           (4 + 5 * 7 ^ 66l / 7 + 890) // violation 'Should use uppercase 'L'.'
                                   & (88l + 78 * 4); // violation 'Should use uppercase 'L'.'
                   long good = (4 + 5 * 7 ^ 66L / 7 + 890) & (88L + 78 * 4); // ok
                   long[] array = {
-                          66l, // violation 'Should use uppercase 'L'.'
-                          66L, // ok
+                      66l, // violation 'Should use uppercase 'L'.'
+                      66L, // ok
                   };
```

Comment:
Formatter didn't intended the code properly, another bug found in the formatter

Reply:
Please find existing issue or create new , they are very responsive.